testsuite/features/secondary/srv_advanced_search.feature

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2022 SUSE LLC.
 # Licensed under the terms of the MIT license.
 
-@scope_monitoring
+@scope_spacewalk_utils
```

Comment:
This is not the correct scope neither, but I don't want to spend time now adding a special one for this feature, and this other seems somehow better on its meaning.